composer.json

```diff
@@ -1,30 +1,88 @@
 {
-    "name": "mautic/language-packager",
-    "description": "Creates language packages for Mautic releases",
-    "license": "GPL-2.0+",
-    "type": "project",
-    "authors": [
-        {
-            "name": "Michael Babker",
-            "homepage": "https://www.mautic.org"
-        }
-    ],
-    "require": {
-        "php": ">=5.5",
-        "joomla/application": "~1.3",
-        "joomla/filesystem": "~1.2",
-        "joomla/http": "~1.1",
-        "joomla/input": "~1.2",
-        "joomla/registry": "~1.4",
-        "babdev/transifex": "~1.0",
-        "aws/aws-sdk-php": "~2.7"
+  "name": "mautic/language-packager",
+  "description": "Creates language packages for Mautic releases",
+  "license": "GPL-2.0+",
+  "type": "project",
+  "minimum-stability": "stable",
+  "prefer-stable": true,
+  "require": {
+    "php": ">=8.1",
+    "ext-ctype": "*",
+    "ext-iconv": "*",
+    "ext-json": "*",
+    "ext-zip": "*",
+    "aws/aws-sdk-php-symfony": "^2.6",
+    "guzzlehttp/guzzle": "^7.5",
+    "http-interop/http-factory-guzzle": "^1.2",
+    "mautic/transifex": "^5.0",
+    "symfony/console": "6.2.*",
+    "symfony/dotenv": "6.2.*",
+    "symfony/flex": "^2",
+    "symfony/framework-bundle": "6.2.*",
+    "symfony/runtime": "6.2.*",
+    "symfony/yaml": "6.2.*"
+  },
+  "require-dev": {
+    "friendsofphp/php-cs-fixer": "^3.16",
+    "phpstan/phpstan": "^1.10",
+    "phpunit/phpunit": "^10.0",
+    "rector/rector": "^0.15.24",
+    "symfony/maker-bundle": "^1.48"
+  },
+  "config": {
+    "allow-plugins": {
+      "php-http/discovery": true,
+      "symfony/flex": true,
+      "symfony/runtime": true
     },
-    "autoload": {
-        "psr-4": {
-            "Mautic\\": "src/"
-        }
+    "bin-dir": "bin",
+    "sort-packages": true
+  },
+  "autoload": {
+    "psr-4": {
+      "App\\": "src/"
+    }
+  },
+  "autoload-dev": {
+    "psr-4": {
+      "App\\Tests\\": "tests/"
+    }
+  },
+  "replace": {
+    "symfony/polyfill-ctype": "*",
+    "symfony/polyfill-iconv": "*",
+    "symfony/polyfill-php72": "*",
+    "symfony/polyfill-php73": "*",
+    "symfony/polyfill-php74": "*",
+    "symfony/polyfill-php80": "*",
+    "symfony/polyfill-php81": "*"
+  },
```

Comment:
This came with Symfony framework so didn't change.
The `symfony/polyfill` packages are designed to provide PHP functions and features that may not be available on all versions of PHP. By specifying this replace section in the composer.json file, the package manager will install the specified versions of the polyfill packages instead of the original packages, if they are required by any other dependencies of the project.

Reply:
It's weird to have polyfills for PHP versions prior to 8.1 as this project requires PHP 8.1. But I guess it won't hurt. I was just curious where it came from.